defuse_wrapper.sh

```diff
@@ -0,0 +1,12 @@
+#!/bin/bash
+set -e
+
+finish() {
+    # Fix ownership of output files
+    uid=$(stat -c '%u:%g' /data)
+    chown -R $uid /data
+}
+trap finish EXIT
+
+# Call tool with parameters
+/home/defuse-0.6.2/scripts/defuse.pl "$@" 
```

Comment:
These should all be in `/usr/local/bin` or `/opt/tool-name/...`. make this change and i'll merge immediately